test/Tests/src/Data/Time/Period_Spec.enso

```diff
@@ -27,9 +29,9 @@ spec =
             (Period.between (Date.new 2022 10 19) (Date.new 2022 11 01)) . should_equal (Period.new days=13)
 
         Test.specify "should not compare between two periods" <|
-            ((Period.new days=10) > (Period.new days=1)) . should_fail_with Incomparable_Values_Error
-            ((Period.new years=10) > (Period.new days=1)) . should_fail_with Incomparable_Values_Error
-            ((Period.new years=10 months=3) > (Period.new months=5)) . should_fail_with Incomparable_Values_Error
+            ((Period.new days=10) > (Period.new days=1)) . should_fail_with Incomparable_Values
+            ((Period.new years=10) > (Period.new days=1)) . should_fail_with Incomparable_Values
+            ((Period.new years=10 months=3) > (Period.new months=5)) . should_fail_with Incomparable_Values
```

Comment:
It's constantly confusing me that this is a 0-constructor 'singleton' type.

By the way - wouldn't it make more sense to extend this error to contain in its payload the two offending values which were unable to be compared? I think it may be easier when debugging to immediately see what kind of objects were attempted to be compared and failed.